### PR TITLE
Update  year LICENSE to 2024

### PR DESCRIPTION
**Pull request by 22388o**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 RGB-Tools developers
+Copyright (c) 2024 RGB-Tools developers
```
Comment (zoedberg) [inline LICENSE:3]:
```suggestion
Copyright (c) 2022-2024 RGB-Tools developers
```

**Review by zoedberg — CHANGES_REQUESTED** [review]
Thanks for the PR. I've requested a small change (please amend the commit instead of adding another one)

File: LICENSE (hunk 1 above)
Comment (22388o) [reply LICENSE:3]: I think is better  :)

**Comment by 22388o** [comment]
@zoedberg I fixed

**Comment by zoedberg** [comment]
@22388o please squash the commits

**Pull request closed without merging**